wsi/overlaybuffer.h

```diff
@@ -50,7 +50,8 @@ class OverlayBuffer {
 
   virtual uint32_t GetFb() const = 0;
 
-  virtual GpuImage ImportImage(GpuDisplay egl_display) = 0;
```

Comment:
Can we have this as GpuDevice and typedef it to EGLDisplay and VkDevice appropriately ? same with GpuImage ?